src/components/Layout/index.tsx

```diff
@@ -1,4 +1,5 @@
 import styles from "./Layout.module.scss";
+import React from 'react';
```

Comment:
NIT: Importing react is not necessary. In Next JS by default we have react imported